### DISCUSSION TITLE
[Feature Request] Storing canvas icons in exported dashboard json

### DISCUSSION THREAD

**Discussion by ptoews** [grafana/grafana#68976] [Dashboards, Panels & Visualizations]
We are using a canvas with custom icons on a dashboard that we want to distribute onto many devices. However, as far as we know, we would have to manually copy the icon files on each instance into the /usr/share/grafana/.../icons path. 

It would be nice if the icons (SVGs in our case) could be integrated into the dashboard JSON, so that we only need to import that.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82705. Feel free to continue the discussion around this there. Thank you!
